### PR TITLE
nodejs-ca: Add Catalan language

### PR DESCRIPTION
**Pull request by franher**
This PR is related to #1197 

I created the minimal set of files according [Node.js Website Translation Policy](https://github.com/nodejs/nodejs.org/blob/master/TRANSLATION.md#for-localization-groups) indications.

* nodejs-ca: Add files - `index.md`, `404.md`, `security.md`, ` site.json`

### CODE REVIEW

**Comment by gtewallace** [comment]
Molt be!!

**Comment by fhemberger** [comment]
Can anyone please check the translation and give a +1?

**Comment by sotayamashita** [comment]
Please review it, if you can. :bow:  - @jseijas, @yagoff

**Comment by franher** [comment]
Hello @sotayamashita, @jseijas and @yagoff are not catalan speakers, so I think they cannot validate my translations. 

@axarrafols is doing the review. I'm gonna fix a couple of typo and I tell him for the final validation.

**Comment by PatrickHeneise** [comment]
Maybe @sgimeno or @masylum can validate?

File: locale/ca/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  tagline-current: Últimes característiques
+  tagline-lts: Recomanat per a la majoria
+  changelog: Canvis
+  api: Documentació de l'API
+  version-schedule-prompt: O revisi la
+  version-schedule-prompt-link-text: Agenda de LTS
+---
+
+Node.js® és un entorn d'execució per a JavaScript construït amb el [motor de JavaScript V8 de Chrome](https://developers.google.com/v8/).
+Node.js fa servir un model d'operacions E/S sense bloqueig i orientat a esdeveniments, que el fa lleuger i eficient.
```
Comment (sgimeno) [inline locale/ca/index.md]: Tiny improvement `... i orientat a esdeveniments, (motiu pel qual | cosa que el fa)`

Comment (franher) [reply locale/ca/index.md]: let me know if you have found something more and then I'll do the final commit :muscle: thanks dude @sgimeno !

File: locale/ca/security.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+de publicació. Un cop el problema és confirmat es determina una llista de totes les versions afectades. El codi és auditat
+per trobar potencials problemes similars. Es preparen les solucions per a totes les versions que estan en manteniment.
+Aquestes solucions no són enviades al repositori públic, en canvi són retingudes localment fins a l'anunci públic.
+
+- Una data d'embargament per a aquesta vulnerabilitat és seleccionada i un CVE (Common Vulnerabilities and  Exposures (CVE®))
+és sol·licitat per la vulnerabilitat.
+
+- A la data d'embargament, se li envia una còpia de l'anunci a la llista de correu de seguretat de Node.js. Els canvis són pujats al repositori públic i noves versions són desplegades en nodejs.org. En les següents 6 hores de la notificació a la llista de correu, una còpia de l'anunci es publicarà al blog de Node.js.
+
+- Normalment la data d'embargament s'establirà 72 hores des de la creació del CVE. Però, això pot variar depenent de
```
Comment (sgimeno) [inline locale/ca/security.md]: Don't need a pause after the conjunction I think: `Però (,)`

File: locale/ca/index.md (hunk 1 above)
Comment (sgimeno) [reply locale/ca/index.md]: added some "super-tiny" improvements, they might even be wrong... All reads good to me, good job 👍

Comment (franher) [reply locale/ca/index.md]: thank you @sgimeno I just committed changes according your indications https://github.com/nodejs/nodejs.org/pull/1221/commits/6b1a8b22960a6b245ebba5fe46d6c6f699d8475b 

If everything looks ok for you, could you give me :+1: ?

File: locale/ca/site.json [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    "conduct": {
+      "link": "https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#code-of-conduct",
+      "text": "Conducta"
+    }
+  },
+  "security": { "link": "security", "text": "Seguretat" },
+  "blog": { "link": "blog", "text": "Notícies" },
+
+  "releases": {
+    "title": "Release History",
```
Comment (andresgutgon) [inline locale/ca/site.json]: Should be this translated to catalan?

Comment (franher) [reply locale/ca/site.json]: sure I missed that 😭

Comment (franher) [reply locale/ca/site.json]: @andresgutgon added missed translation on a2afa18

**Comment by sotayamashita** [comment]
@franher Oops, sorry. By the way, is there `nodejs-ca` team. I cannot find that ?

**Comment by franher** [comment]
@sotayamashita no problem :) 

As soon I know, there is not Node.js Catalan team.

**Comment by sgimeno** [comment]
What's the purpose of these teams?

**Comment by franher** [comment]
@fhemberger hello :) I think some catalan colleagues have already done the review and have given +1

**Comment by fhemberger** [comment]
Cool, thanks for your contribution!

**Comment by sotayamashita** [comment]
@sgimeno I think local teams are responsible for management local translation. When someone translates docs into the Catalan language, we need to review but if the team does not exist, it could be very hard to find people can review.  Does it make sense?

**Comment by sgimeno** [comment]
@sotayamashita sure. Does it make sense to build that team now for future reviews and contributions? I'm up for it

**Comment by franher** [comment]
@sgimeno I'm available too :)

**Comment by albertvp** [comment]
@sgimeno @franher count on me!

File: locale/ca/site.json [hunk 4]
```diff
@@ -0,0 +1,177 @@
+{
+  "title": "Node.js",
+  "author": "Fundació Node.js",
+  "url": "https://nodejs.org/ca/",
+  "locale": "ca",
+  "scrollToTop": "Torna al començament",
+  "reportNodeIssue": "Informe d'un problema de Node.js",
+  "reportWebsiteIssue": "Informe d'un problema al lloc web",
+  "getHelpIssue": "Aconseguir ajuda",
```
Comment (albertvp) [inline locale/ca/site.json:9]:
```
"scrollToTop": "Tornar al començament",
"reportNodeIssue": "Informi d'un problema de Node.js",
"reportWebsiteIssue": "Informi d'un problema al lloc web",
"getHelpIssue": "Ajuda",
```
This sounds better to me... @franher

**Comment by albertvp** [comment]
The links on navigation menu are broken... maybe there's any missed file?

File: locale/ca/site.json (hunk 4 above)
Comment (franher) [reply locale/ca/site.json:9]: great @albertvp the PR is already landed in, so I invite you to create new one adding this modifications and more translations (the remaining files)

Comment (albertvp) [reply locale/ca/site.json:9]: Ok, I'll check them in detail

**Pull request merged**